asahi-desktop-meta/PKGBUILD

```diff
@@ -11,7 +11,7 @@ install=asahi-desktop-meta.install
 
 package() {
   # Put depends in package() to avoid unecessary build dependencies
-  depends=(bluedevil bluez-utils bluez-tools pipewire pipewire-audio pipewire-pulse pipewire-alsa wireplumber)
+  depends=(bluedevil bluez-utils bluez-tools pipewire pipewire-audio pipewire-pulse pipewire-alsa wireplumber asahi-audio bankstown speakersafetyd)
```

Comment:
Shouldn't you also bump the `pkgrel` and/or `pkgver` when modifying the dependencies? (Not 100% sure what's best practive in Arch here, just my guess)

Reply:
that may be a good idea, yes